CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.9.1 (2018-04-25)
+
+* Lazy load asyncio.Lock on redis backend. This fixes an issue with multiprocess
+apps like tornado that might fork after import time. - Jordi Soucheiron
```

Comment:
No need to add this, the changelog is generated automatically whenever there is a new release